webserver/views/api/v1/core.py

```diff
@@ -182,10 +186,15 @@ def get_data_for_multiple_recordings(collect_data):
     recordings = check_bad_request_for_multiple_recordings()
 
     recording_details = {}
-
+    map_keys = request.args.get("map")
     for recording_id, offset in recordings:
         try:
-            recording_details.setdefault(recording_id, {})[offset] = collect_data(recording_id, offset)
+            if map_keys:
+                recording_details.setdefault(recording_id, {})[offset] = collect_data(
```

Comment:
there's a bug here if someone calls the lowlevel get multiple function and provides a `map` query parameter. It will try and call `load_low_level` with a `map_keys` parameter, which will fail

Reply:
Now we don't have to perform this check, as `load_high_level` performs how it used to, now mapping will be done via a pre-processing step as you suggested